guides/source/active_job_basics.md

```diff
@@ -12,6 +12,7 @@ After reading this guide, you will know:
 * How to enqueue jobs.
 * How to run jobs in the background.
 * How to send emails from your application asynchronously.
+* How to halt a job's lifecycle callback execution.
```

Comment:
We don't mention callbacks at all, so why should we mention halting them?)
If we are going to add this, it probably should be more generic:
```suggestion
* How to use callbacks in a job's lifecycle.
```
But I'm inclined to suggest just removing it, as it's not a big part of jobs and there are a lot more concept we don't mention here either  🤷‍♂️